src/main/java/edu/uc/javahotkey/JavaHotKeyController.java

```diff
@@ -25,11 +26,8 @@ public Project sampleJsonSchema() {
         project.setName("Sample Project");
         KeyMap keyMap = new KeyMap();
         keyMap.setFunctionName("func_name");
-        keyMap.setKeymap(new ArrayList<Integer>());
-        keyMap.getKeymap().add(17);
-        keyMap.getKeymap().add(65);
-        project.setKeymaps(new ArrayList<KeyMap>());
-        project.getKeymaps().add(keyMap);
+        keyMap.setKeymap(new ArrayList<>(List.of(17, 65)));
+        project.setKeymaps(new ArrayList<>(List.of(keyMap)));
```

Comment:
I find this type of declaration easier to read than the previous format. If the lists were longer, the difference would be more pronounced because you could see the whole list together instead of having a separate line for each list item.

Reply:
This is really up to the programmer.  My preference would be to keep it as it were.  While it's nice to consolidate multiple lines into one, the consequence is, it's also difficult to determine the source of an error when you have one line number listed in the error, and there are several method calls and/or operations occurring.